distribution/sdk/sample-nsili-client/src/main/java/org/codice/alliance/nsili/client/Client.java

```diff
@@ -126,31 +132,29 @@ public void runTest(String[] args) throws Exception {
             server.stop();
         }
 
-        System.out.println("Done. ");
+        LOGGER.info("Done. ");
         System.exit(0);
     }
 
     public static void main(String args[]) throws Exception {
         Client client = new Client();
         if (args.length != 1) {
-            System.out.println("Unable to obtain IOR File :  Must specify URL to IOR file.");
+            LOGGER.info("Unable to obtain IOR File :  Must specify URL to IOR file.");
         }
         client.runTest(args);
```

Comment:
Outside the scope of this PR, but probably should have a LOGGER.error() for the case that the client cannot connect to the server. `Error recieving IOR File`